src/components/Card/Card.css

```diff
@@ -92,6 +96,14 @@
   .card {
     margin: 0;
   }
+
+  .card__title_type_cards-with-images {
+    min-height: 0;
+  }
+
+  .card__text_type_team {
+    min-height: 0;
+  }
```

Comment:
it looks like `card__text_type_team` got added twice with diff min-heights

Reply:
Actually, one of those is inside of a media query. It didn't look like it because that part of the code was folded.

![image](https://user-images.githubusercontent.com/37159177/112761496-966f3100-8fc9-11eb-9774-a6f9871ec35d.png)